src/Faker/Provider/Person.php

```diff
@@ -123,4 +123,14 @@ public static function titleFemale()
     {
         return static::randomElement(static::$titleFemale);
     }
+
+    /**
+     * @example 'female'
+     */
+    public function gender()
```

Comment:
Useless IMO, and unrelated to this PR anyway

Reply:
If you check the getters for salutation and nameInformal you can put in the gender as parameter (optional). I've added this, because I was generating a csv file and I needed values for salutation and nameInformal in 2 columns and in that case both columns should have the same gender in 1 row:

```php
for($i = 1; $i <= 100; $i++)
{

    $gender = $faker->gender;
    $anreden = ['male' => 'Herr', 'female' => 'Frau'];

    $birthday = $faker->optional(0.9)->dateTimeThisCentury('-18 years');
    $birthday = !is_null($birthday) ? $birthday->format('Y-m-d') : '';

    $validityBegin = $faker->dateTimeBetween('-11 months','+1 month');
    $validityEnd = clone $validityBegin;
    $validityEnd = $validityEnd->add(\DateInterval::createFromDateString('P1Y'));

    $rawPrice = $faker->optional(0.9)->randomFloat(2,0,2000);
    $price = !is_null($rawPrice) ? number_format($rawPrice,2) : '';

    fputcsv(
      $handle,
      [
        $i,
        $validityBegin->format('Y-m-d H:i:s'),
        $validityEnd->format('Y-m-d H:i:s'),
        $prefix.$faker->randomNumber(6),
        $faker->optional(0.9)->randomElement(['a', 'b', 'c']),
        $faker->optional(0.9)->date('Ymd'.str_pad($faker->numberBetween(0,10000), 5, '0', STR_PAD_LEFT)),
        $faker->optional(0.9)->randomNumber(2),
        $price,
        $faker->unique()->randomNumber(8),
        $faker->email,
        $anreden[$gender],
        $faker->optional(0.1)->title,
        $faker->optional(0.1)->company,
        $faker->optional(0.9)->firstName($gender),
        $faker->optional(0.9)->lastName,
        $birthday,
        $faker->optional(0.9)->streetName,
        $faker->optional(0.9)->postcode,
        $faker->optional(0.9)->city,
        $faker->optional(0.9)->countryCode,
        $faker->optional(0.7)->phoneNumber,
        $faker->optional(0.9)->numberBetween(1,2),
        $faker->optional(0.9)->text(255),
        $faker->optional(0.9)->randomElement(['a', 'b'])
      ],
      ';',
      '"'
    );
}
```

I though it was convenient to have a getter for the gender in Faker, but I can remove it again if you wish to.